src/js/_store/modules/categories.js

```diff
@@ -54,6 +67,24 @@ export default {
         commit('failRequest', { message: err.message });
       });
     },
+    setDeleteCategoryInfo({ commit }, { id, name }) {
+      const payload = {
+        categoryId: id,
+        categoryName: name,
+      };
+      commit('setDeleteCategoryInfo', payload);
+    },
+    deleteCategory({ commit, rootGetters }) {
+      axios(rootGetters['auth/token'])({
+        method: 'DELETE',
+        url: `/category/${rootGetters['categories/deleteCategoryId']}`,
```

Comment:
```suggestion
    deleteCategory({ commit, rootGetters, state }) {
      axios(rootGetters['auth/token'])({
        method: 'DELETE',
        url: `/category/${state.deleteCategoryId}`,
```

上記の書き方でいけるので試してみてください！

同一storeのstateを参照したいだけなのでわざわざgettersを見なくても大丈夫です！

Reply:
修正しました。
15240dd230cda12a7a146fee4ba5e585f4b860d7